community/modules/scheduler/schedmd-slurm-gcp-v5-hybrid/README.md

```diff
@@ -0,0 +1,204 @@
+## Description
+This module is a wrapper around the [slurm-controller-hybrid] module by SchedMD
+as part of the [slurm-gcp] github repository. The hybrid module serves to create
+the configurations needed to extend an on-premise slurm cluster to one with one
+or more Google Cloud bursting partitions. These partitions will create the
+requested nodes in a GCP project on-demand and scale after a period of not being
+used, in the same way as the [schedmd-slurm-gcp-v5-controller] module
+auto-scales VMs.
+
+> **_NOTE:_** This module has only been tested in limited capacity with the HPC
+> Toolkit. On Premise configurations can vary significantly, this module should
+> be used as a starting point, not a complete solution.
```

Comment:
I would explicitly add that this module is `experimental` and that it is expected that the module and documentation may change significantly in the future. 

Also add it to the list of modules, that may have gotten lost in the refactor.

Reply:
Done